src/connector/src/parser/unified/json.rs

```diff
@@ -483,9 +485,17 @@ impl JsonParseOptions {
             (
                 DataType::Timestamp,
                 ValueType::I64 | ValueType::I128 | ValueType::U64 | ValueType::U128,
-            ) => i64_to_timestamp(value.as_i64().unwrap())
-                .map_err(|_| create_error())?
-                .into(),
+            ) => {
+                if let TimestamptzHandling::Milli = self.timestamptz_handling {
```

Comment:
@xiangjinwu is it a good practice to use the `timestamptz_handling` for timestamp handling? The naming seems to suggest the it is related to tz.

Reply:
cc @xiangjinwu WDYT?